src/pages/SeriesDetailsPage.js

```diff
@@ -15,14 +15,15 @@ const {
 export default function SeriesDetailsPage() {
   const { id } = useParams();
   const seriesDetailsUrl = `${REACT_APP_API_BASE_SERIES_URL}/${id}?api_key=${REACT_APP_API_KEY}&language=${REACT_APP_API_LANGUAGE}`;
-  const { data: series, loading: isLoading } = useDetails(seriesDetailsUrl);
+  const { data: series, isLoading } = useDetails(seriesDetailsUrl);
+  const navigate = useNavigate();
 
   return (
     <Wrapper>
-      <StyledLinkBack to="/serien">
+      <StyledButtonBack onClick={goBack}>
```

Comment:
also here you could change it to an arrow function and use it directly instead of the additional function😊